pkgs/development/interpreters/python/mk-python-derivation.nix

```diff
@@ -252,6 +252,7 @@ let
       #    because the hook that checks for conflicts uses setuptools.
       #
       pythonCatchConflictsHook
+    ] ++ optionals (attrs ? pythonRelaxDeps || attrs ? pythonRemoveDeps) [
```

Comment:
You are just adding an empty list here? Looks like the tree-wide change in the last commit might have mistakenly deleted pythonRelaxDepsHook here? :D

Reply:
Thanks for the review!